lib/elixlsx/sheet.ex

```diff
@@ -219,4 +221,9 @@ defmodule Elixlsx.Sheet do
   def remove_pane_freeze(sheet) do
     %{sheet | pane_freeze: nil}
   end
+
+  @spec add_validation(Sheet.t(), String.t(), String.t(), list(String.t())) :: Sheet.t()
```

Comment:
Nit: Would suggest renaming _validations to _data_validations, in line with the XLST naming and to make it a bit more explicit what is being validated, WDYT?

Reply:
You are right, I agree that  `_data_validations` is a better name. Changes done!

Reply:
@xou can you take a look again, please?